federation/pkg/federation-controller/replicaset/replicasetcontroller.go

```diff
@@ -142,7 +142,7 @@ func NewReplicaSetController(federationClient fedclientset.Interface) *ReplicaSe
 			&extensionsv1.ReplicaSet{},
 			controller.NoResyncPeriodFunc(),
 			fedutil.NewTriggerOnAllChanges(
-				func(obj runtime.Object) { frsc.deliverLocalReplicaSet(obj, replicaSetReviewDelay) },
+				func(obj runtime.Object) { frsc.deliverReplicaSetObj(obj, replicaSetReviewDelay) },
```

Comment:
Yes: the only conceptual difference between `deliverLocalReplicaSet` and `deliverFedReplicaSet` was that `deliverLocalReplicaSet` checked for the existence of the replica set before starting reconciliation. Since reconcile does that anyway, this extra code complexity isn't worth it IMO.